generate-stackbrew-library.sh

```diff
@@ -90,6 +90,9 @@ for version in "${versions[@]}"; do
 		case "$version" in
 		        19|18) variantArches=( ${variantArches[@]/ppc64le} )
 		esac
+		case "$variant" in
+			jessie) variantArches=( amd64 arm32v7 i386 )
+		esac
```

Comment:
I feel `amd64` is no problem we did that before, but wonder did you test `arm32v7` `i386` or from where you know it's working

Reply:
https://github.com/docker-library/official-images/blob/master/library/buildpack-deps#L62-L65
I copy the architectures from the page.

Reply:
that line has defined 4 architectures instead of 3; additionally, the base jessie can run on arm32v7 doesn't mean erlang code can compile well with `arm32v7`; and the sometime before I remember people report erlang19 (or one of previous versions) couldn't compile on another arch (ppc or another)
so: I mean unless you have that architecture hardware (or emulator) proved it indeed compile and run well,
I am intended not to include others than `amd64`

and I believe somewhere in mid of 2019 we will eventually delete all jessie related support, that's what we agreed on the original PR who requested for jessie support; we would rather do less than more, for those already deprecated stuff

Only the version bump PRs we can merge fast,

for others (include the rebar3 PR as well), please have some patience, I'd like to do some more local testing. (pending time for another dev may report some potential problem, or tell us it indeed works on another architecture ...),
the whole process won't be too long.